blocks/newsletter-form/block.json

```diff
@@ -0,0 +1,72 @@
+{
+	"apiVersion": 2,
+	"name": "cata/newsletter-form",
```

Comment:
I think `newsletter-form` is longer than we need. Just newsletter is enough. That should make the autogenerated class name shorter too.

Reply:
updated all instances of `newsletter-form` to `newsletter`